components/Ripple/src/MDCRippleTouchController.h

```diff
@@ -56,17 +56,45 @@
 @property(nonatomic, assign) BOOL shouldProcessRippleWithScrollViewGestures;
 
 /**
- Unavailable, please use `initWithView` instead.
+ Initializes the controller.
+
+ @param view The view that responds to the touch events for the ripple, and the view which the
+ ripple is added as a subview to.
  */
-- (nonnull instancetype)init NS_UNAVAILABLE;
+
+/**
+ Initializes the controller and adds the initialized ripple view as a subview of the provided view.
+
+ Note: When using this initializer, calling `addRippleToView:` isn't needed.
+
+ @param view The view that responds to the touch events for the ripple, and the view which the
+ ripple is added as a subview to.
```

Comment:
Nit/grammar: "...which the ripple is added as a subview to" sounds very awkward.  Maybe something like 

```
@param The view that responds to the touch events for the ripple. The ripple 
       is added to it as a subview.
```

Reply:
Done